package.json

```diff
@@ -11,7 +11,7 @@
 		"@microsoft/sarif-web-component": "^0.6.0-24",
 		"@sinonjs/text-encoding": "^0.7.1",
 		"applicationinsights-js": "^1.0.21",
-		"azure-devops-extension-api": "^1.153.2",
+		"azure-devops-extension-api": "^1.158.0",
```

Comment:
How'd you identify that 158 contained the fix?

Reply:
Just looked at the history of the code I'm calling.